DependencyInjection/Configuration.php

```diff
@@ -91,7 +91,11 @@ private function addDbalSection(ArrayNodeDefinition $node) : void
                             ->end()
                             ->children()
                                 ->scalarNode('class')->isRequired()->end()
-                                ->booleanNode('commented')->setDeprecated()->end()
+                                ->booleanNode('commented')->setDeprecated(
+                                    'doctrine/DoctrineBundle',
```

Comment:
We should use the packagist name:
```suggestion
                                    'doctrine/doctrine-bundle',
```